lib/plugins/aws/package/compile/events/apiGateway/lib/stage.test.js

```diff
@@ -59,7 +59,7 @@ describe('#compileStage()', () => {
       };
     });
 
-    it('should create a dedicated stage resource if tracing is configured', () =>
+    it.skip('should create a dedicated stage resource if tracing is configured', () =>
```

Comment:
Why skip? Is there a plan to make it working later?

Reply:
Yes, exactly (see comment below)